scripts/verdaccio.sh

```diff
@@ -18,12 +18,10 @@ function cleanup {
   then
     lsof -ti tcp:4000 | xargs kill
     # Clean up generated dists if run locally
-    shopt -s globstar extglob
```

Comment:
only intel macs need this, m macs will throw on this and crash, failing to cleanup. now that the majority of the common users of this script are on m macs, I'm removing it